packages/webdriverio/src/scripts/customElement.ts

```diff
@@ -17,13 +17,13 @@ export default function customElementWrapper () {
                 parentNode = parentNode.parentNode
             }
             console.debug('[WDIO]', 'newShadowRoot', this, parentNode, parentNode === document)
-            return origConnectedCallback.call(this)
+            return origConnectedCallback && origConnectedCallback.call(this)
         }
 
         const origDisconnectedCallback = Constructor.prototype.disconnectedCallback
         Constructor.prototype.disconnectedCallback = function(this: HTMLElement) {
             console.debug('[WDIO]', 'removeShadowRoot', this)
-            return origDisconnectedCallback.call(this)
+            return origDisconnectedCallback && origDisconnectedCallback.call(this)
```

Comment:
let's use modern js syntax
```suggestion
            return origDisconnectedCallback.call?.(this)
```

Reply:
Same as above would need to be changed to 
`return origConnectedCallback?.call(this)`